src/main/java/com/adyen/Util/Util.java

```diff
@@ -155,4 +156,16 @@ public static String calculateSessionValidity() {
 
         return fmt.format(sessionDate);
     }
+
+    /**
+     * Encode the given string to base 64 string
+     * @param input string
+     * @return base 64 encoded string    "
+     */
+    public static String toBase64Encode(String input) {
+        if (input != null) {
```

Comment:
should we not just do it. If it fails the exception is fine.

Reply:
The method is moved to SaleToAcquirerData class